pkg/controller/limited-support/limited_support_test.go

```diff
@@ -0,0 +1,119 @@
+package limited_support
```

Comment:
Can we change the file name to `limited_support_controller_test.go` to match the pattern of this repo?

Reply:
Absolutely. Good catch!